Makefile

```diff
@@ -72,8 +66,8 @@ $(GOVERALLS_BIN):
 	go get -u $(GOVERALLS_PKG)
 
 $(LINT_BIN):
-	@$(call print, "Fetching gometalinter.v2")
-	GO111MODULE=off go get -u $(LINT_PKG)
+	@$(call print, "Fetching linter")
+	go get -u $(LINT_PKG)
```

Comment:
The current method of installation ends up producing a `go.mod` diff when running `make lint`. This is probably because `go get` thinks we want to add this as a dependency to the project rather than just install the binary. We can avoid this by prepending `GO111MODULE=off`.

Reply:
You're right, good catch!